src/test/routes/delete.user.profile.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { app } from "../../main/app";
+import { appTest } from "../../main/app.test";
+import { expect } from "chai";
+import { get } from "config";
+import * as sinon from "sinon";
+import * as idamServiceMock from "../http-mocks/idam";
+import * as mock from "nock";
+import * as request from "supertest";
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+describe("Confirm Delete page", () => {
+    beforeEach(() => {
+        const config = {
+            get: sinon.stub(),
+        };
+        config.get.withArgs("adminWeb.userprofiles_url").returns("http://localhost:4453/users");
+    });
+
+    describe("on GET /deleteuserprofile", () => {
+
+        it("should redirect to import page when not authenticated", () => {
+            return request(app)
+                .get("/deleteuserprofile")
+                .then((res) => {
+                    expect(res.statusCode).to.equal(302);
+                    expect(res.headers.location.startsWith(get("adminWeb.login_url"))).to.be.true;
+                });
+        });
+
+        it("should return Confirm delete user profile page when authenticated", () => {
+            idamServiceMock.resolveRetrieveUserFor("1", "admin");
+            idamServiceMock.resolveRetrieveServiceToken();
+
+            return request(app)
+                .get("/deleteuserprofile")
+                .query({ idamId: "anas@yahoo.com" })
+                .set("Cookie", "accessToken=ey123.ey456")
+                .then((res) => {
+                    expect(res.statusCode).to.equal(200);
```

Comment:
The correct HTTP response code should be a `204` when there is no content to return